python/node_classification.py

```diff
@@ -0,0 +1,641 @@
+from torch_geometric.data import Data
+from mage.node_classification.models.inductive_model import InductiveModel
+from mage.node_classification.models.gatjk import GATJK
+from mage.node_classification.utils.metrics import metrics
+from mage.node_classification.utils.extract_from_database import extract_from_database
+from mage.node_classification.models.train_model import train_epoch
+import mgp
+from tqdm import tqdm
+import torch
+import numpy as np
+import os
+from torch_geometric.nn import to_hetero
+import typing
+
+
+##############################
+# constants
+##############################
+
+# parameters for the model
+class ModelParams:
+    IN_CHANNELS = "in_channels"
+    OUT_CHANNELS = "out_channels"
+    HIDDEN_FEATURES_SIZE = "hidden_features_size"
+    LAYER_TYPE = "layer_type"
+    AGGREGATOR = "aggregator"
+
+
+# paramters for type of layers inside model
+class LayerType:
+    gat = "GAT"
+    gatv2 = "GATv2"
+    sage = "SAGE"
+    gatjk = "GATJK"
```

Comment:
What is GATJK?

Reply:
GAT with jumping knowledge. Great model for fraud detection.